client/src/plus/collections/v2/collection.tsx

```diff
@@ -23,21 +28,30 @@ export default function CollectionComponent() {
     mutate,
   } = useItems(collectionId);
 
+  const navigationType = useNavigationType();
+  useEffect(() => {
+    if (navigationType === "PUSH") document.documentElement.scrollTo(0, 0);
+  }, [navigationType]);
+
   return collection ? (
     <>
-      <header className="plus-header">
+      <header>
         <Container>
-          <Link to="../">Exit Collection</Link>
+          <Link to="../" className="exit">
+            &larr; Exit
+          </Link>
           <h1>{collection.name}</h1>
-          <p>{collection.description}</p>
+          <span className="count">
+            {collection.article_count}{" "}
+            {collection.article_count > 0 ? "articles" : "article"}
```

Comment:
```suggestion
            {collection.article_count === 1 ? "article" : "articles" }
```
Otherwise you get '0 article' , '1 articles'